frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -1,23 +1,24 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
-import { sortable } from '@patternfly/react-table';
```

Comment:
Nit: it seems there are only import-related changes in this file, so maybe skip them unless necessary.